hazelcast/src/main/java/com/hazelcast/sql/impl/SqlDataSerializerHook.java

```diff
@@ -201,10 +202,9 @@
     public static final int EXPRESSION_EXTRACT = 72;
 
     public static final int MAP_SCAN_METADATA = 73;
-//    Reserved
-//    public static final int MAP_INDEX_SCAN_METADATA = 74;
+    public static final int MAP_INDEX_SCAN_METADATA = 74;
 
-    public static final int LEN = MAP_SCAN_METADATA + 1;
+    public static final int LEN = MAP_INDEX_SCAN_METADATA + 1;
```

Comment:
This is confusing, some parts of Map/Map Index scanning serialization are defined here, other in `JetSqlSerializerHook`.

Reply:
Processors serialization defined in `JetSqlSerializerHook`, everything else in `SqlSerializerHook`

Reply:
Metadata classes are used solely by custom IMap processors. They could be moved to the same package, access modifiers adjusted and serialization configuration could be placed in single hook.